tchannel/event.py

```diff
@@ -35,6 +36,7 @@
     'EventType',
     before_send_request=0x00,
     after_send_request=0x01,
+    before_serialize_request_headers=0x02,
```

Comment:
wait, this should be "send", disregard other comments.

Reply:
why send?

Reply:
ah, because other things call it with "send".  initially it was send, but I voted to change it to "serialize" since all comments around it / its purpose was for before-serialization.

unless there's a reason to prefer "send" in the event name I'll vote for the calls to be "serialize" too.

Reply:
yeah, sgtm